commcare_connect/templates/tailwind/pages/dashboard.html

```diff
@@ -0,0 +1,31 @@
+<!-- dashbaord.html -->
+{% extends 'ccc_app/base.html' %}
+
+{% block content %}
+
+<div id="content-area" class="md:max-w-screen-2xl mx-auto flex items-center flex-row gap-4 min-h-screen"> 
+    <!-- Left Column -->
+    <div class="w-2/3" >
+       <h2 class="mb-4 text-brand-deep-purple font-medium text-sm">Programs</h2>
+       <div class="flex flex-col gap-4">
+         {% for program in data.programs %}
```

Comment:
All the pages you have inside pages directory should be in their respective django app dirs.

This file -> templates/program/tailwind/dashboard.html
opportunities -> templates/opportunity/tailwind/
worker -> templates/users/tailwind/

Reply:
Also, as I pointed out earlier, these should be using the files I have already created https://github.com/dimagi/commcare-connect/compare/main...sr/program-ui#diff-47aaa001022d02052de8a8953acdf3c0e0f7dea6199aa7e6cc670d934b707113R1 So that the data inside comes from the view